.gitignore

```diff
@@ -5,5 +5,5 @@
 .remake/
 1_fetch/tmp/*
 !1_fetch/tmp/state_splits.yml
-3_visualize/out
+3_visualize/out/
```

Comment:
Curious what your motivation was for this change. Also, I'm sorta confused about trailing slash vs no slash in .gitignore files - any chance you're able to explain that to me?

Reply:
I don't think I fully understand it either. I was just double checking and saw it didn't have a slash, so I added it, based on what I'd read [here](https://www.quora.com/What-is-the-syntax-to-ignore-a-entire-folder-in-a-gitignore-file). Sounds like the / ensures that the folder by that name is ignored, while without the slash any folder _or_ file by that name would be ignored?

Reply:
Ahhhh, thanks!